go/okapi/native_call.go

```diff
@@ -0,0 +1,109 @@
+package okapi
+
+import (
+	"github.com/coreos/pkg/dlopen"
+	"unsafe"
+)
+
+// #cgo LDFLAGS: -ldl
```

Comment:
Is this comment needed?

Reply:
Yes, actually. This is the `dlopen` `dlsym` kernel extensions.